spec/services/github_pull_request_service_spec.rb

```diff
@@ -195,9 +195,59 @@ def build(*_args)
     end
   end
 
+  describe '#open_prs' do
+    let(:owner) { 'someUser' }
+    let(:reviewer) { create(:github_user, id: 17) }
+    let(:repository) { create(:repository, tracked: true) }
+    let(:pull_request) do
+      create(
+        :pull_request,
+        id: 3,
+        gh_id: 3,
+        title: "Test",
+        repository: repository,
+        pr_state: 'open'
+      )
+    end
+    let(:pull_request_review_request) do
+      create(
+        :pull_request_review_request,
+        pull_request_id: pull_request.id,
+        github_user_id: reviewer.id
+      )
+    end
+
+    context 'when owner has a closed pr saved as open' do
+      it 'returns empty list' do
+        expect(service.open_prs(owner)).to eq([])
+      end
+    end
+  end
+
   describe "#handle_webhook_event" do
     let(:repository) { create(:repository, tracked: true) }
 
+    context 'when event datas action is review_requested_removed' do
+      let(:event_request_data) do
+        double(
+          action: 'review_requested_removed',
+          pull_request: github_pr_response,
+          requested_reviewer: requested_reviewer,
+          repository: double(id: repository.gh_id)
+        )
+      end
+      let(:pull_request) do
+        create(:pull_request, gh_id: 3, title: "Test", repository: repository)
+      end
+
+      it 'calls remove_reviewer_from_pull_request!' do
+        expect(service).to receive(:remove_reviewers_from_pull_request!)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>RSpec/MessageSpies: Prefer have_received for setting message expectations. Setup service as a spy using allow or instance_spy.